ui-frontend/packages/catalog-ui-search/src/main/webapp/react-component/location/base.line.js

```diff
@@ -125,7 +135,6 @@ const BaseLine = (props) => {
   return (
     <div>
       <div className="input-location">
-        <Conversion value={currentValue} isValid={baseLineError} />
```

Comment:
📝  Gets rid of the conversion component in favor of letting the users input the usng/mgrs, dms, and utm/ups.